test/integration/homepage_popular_links_test.rb

```diff
@@ -51,6 +51,48 @@ class HomepagePopularLinksTest < JavascriptIntegrationTest
     end
   end
 
+  context "#confirm_destroy" do
+    setup do
+      click_button("Create new edition")
+    end
+
+    should "show the 'Delete draft' confirmation page with the option to actually delete" do
```

Comment:
can we rename the test to something like 'show the 'Delete draft' confirmation page with the 'Delete' button', this is what it looks like we were trying to test

Reply:
This has been updated to 'show the 'Delete Draft' confirmation page' - not sure why it isn't showing here.